chain/force-upgrade/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+use anyhow::format_err;
+
+use starcoin_types::{
+    account::{Account, DEFAULT_MAX_GAS_AMOUNT},
+    transaction::SignedUserTransaction,
+};
+
+use include_dir::{include_dir, Dir};
+use starcoin_types::account::DEFAULT_EXPIRATION_TIME;
+use starcoin_vm_types::{
+    account_config::STC_TOKEN_CODE_STR,
+    genesis_config::ChainId,
+    transaction::{Package, RawUserTransaction, TransactionPayload},
+};
+
+pub const FORCE_UPGRADE_PACKAGE: Dir = include_dir!("package");
+
+pub struct ForceUpgrade;
+
+impl ForceUpgrade {
+    ///
+    /// block_timestamp: *NOTE* by seconds,
+    pub fn force_deploy_txn(
+        account: Account,
+        sequence_number: u64,
+        block_timestamp: u64,
+        chain_id: &ChainId,
+    ) -> anyhow::Result<SignedUserTransaction> {
+        let package_file = "stdlib.blob".to_string();
+        let package = FORCE_UPGRADE_PACKAGE
```

Comment:
The `package` which has been restored has an empty init_script which is `None` here. Do we need set the `init_script` function explicitly here? If so, when executing the txn to deploy the package, StarcoinVM will call this init_script. The init_script shoud be 0x1::StdlibUpgradeScripts::upgrade_from_v11_to_v12

Reply:
I think we should have a test ensure package has init_script shoud be 0x1::StdlibUpgradeScripts::upgrade_from_v11_to_v12